packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ComponentsElement,
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import {
+  AnnotationElement,
+  toValue,
+  cloneShallow,
+  isObjectElement,
+} from '@swagger-api/apidom-core';
+import { isComponentsElement } from '@swagger-api/apidom-ns-openapi-3-0';
```

Comment:
```suggestion
import { isComponentsElement } from '@swagger-api/apidom-ns-openapi-3-1';
```

Our source is OpenAPI 3.1.0 and not OpenAPI 3.0.x.

Reply:
Ah. Automatic import suggestion fails me once again. I should really double check those 😅